inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java

```diff
@@ -59,12 +66,24 @@ public void afterPropertiesSet() {
         if (storeConfig.isElasticsearchStore()) {
             esService.startTimerRoutine();
         }
+        mqConsume.start();
+    }
 
-        if (mqConsume != null) {
-            mqConsume.start();
-        } else {
-            LOG.error("fail to auditMsgConsumerServer");
+    /**
+     * getInsertServiceList
+     * @return
+     */
+    private List<InsertData> getInsertServiceList() {
+        List<InsertData> insertServiceList = new ArrayList<>();
+        if (storeConfig.isMysqlStore()) {
```

Comment:
may storeConfig matches multi store types? If not, **_swtich case_** may be a better format